problems/calcAvg.yaml

```diff
@@ -0,0 +1,35 @@
+id: 9
+title: CalcAvg
+text: |
+  Write a function called calcAvg that takes in an array of integers
+  and returns the average of all of the integers in the array.
+
+  This function should work with negative numbers!
+  
+  Refer to example below.
+  
+  calcAvg([5, 123, 12, 24]) = 41
+  calcAvg([-12, 23, 45, 67, -89]) = 6.8
+code: |
+  function calcAvg(arr) {
+    return num;
+  }
+
+  assert(calcAvg([5, 123, 12, 24]) === 41);
+  assert(calcAvg([-12, 23, 45, 67, -89]) === 6.8);
+  assert(calcAvg([0, 789, 456, 123, -159]) === 241.8);
+  alert('Passed all test cases!');
+tests:
+  - assert(calcAvg([5, 123, 12, 24]) === 41);
```

Comment:
include edge test cases
e.g. 0 and 1 element.

Reply:
I will ask the user to return null

Reply:
Will do :)